src/components/schedule-page/table/schedule/schedule-parts/shift-cell/shift-cell.component.tsx

```diff
@@ -100,73 +104,97 @@ export function ShiftCellComponentF(options: ShiftCellOptions): JSX.Element {
     ),
     [isBlocked, onClick]
   );
+  let clearModal;
 
-  //  #region view
+  //  #region view?
   return (
-    <td
-      ref={mergeRefs([selectableItemRef, componentContainer])}
-      className={classNames("mainCell", {
-        selection: isSelected || (isComponentVisible && isBlocked),
-        blocked: isBlocked,
-      })}
-      onClick={(): void => toggleComponentVisibility()}
-      id={id}
-      onBlur={(): void => {
-        onBlur?.();
-      }}
-    >
-      {showInput && (
-        <WrapContentDiv>
-          <CellInput
-            input={ShiftAutocompleteComponent}
-            onKeyDown={onKeyDown}
-            onValueChange={_onValueChange}
-            {...options}
-          />
-        </WrapContentDiv>
+    <>
```

Comment:
Super pomysł!
 
Jedyne co, to jest problem z `isComponentVisible`:
w funkcji `onMouseLeave`, ty ustawiasz tę flagę na false. W swojej kolei, kiedy ty pokazujesz ekran, to on przesłania komórkę, przez co  jest trigerowana funkcja `onMouseLeave` i ekran znika po 444 sekundach i a z nim również pożądany pozwalający na odkliknięcie

![image](https://user-images.githubusercontent.com/21079319/111546350-d3d8f080-8777-11eb-957c-9e585a844c49.png)


Moja propozycja jest:
ablo usunąć stąd tę flagę, albo zwiększyć z-index na komórce